README.md

```diff
@@ -285,4 +285,4 @@ ambiguity between variants, the earliest variant will be parsed.
 
 ## Using a custom allocator
 
-Bauble has the capability to use a custom allocator through the `BaubleAllocator` trait. With this you can for example allocate values within an arena.
\ No newline at end of file
+Bauble has the capability to use a custom allocator through the `BaubleAllocator` trait. With this you can for example allocate values within an arena.
```

Comment:
What changed here?

Reply:
I guess end of file newline git adds

Reply:
Ah okay, in that case you should be good to merge.